### ISSUE TITLE
Don't require sudo as part of installation process if run as root

### ISSUE THREAD

**Issue by QuingKhaos** [terraform-linters/tflint#1101]
The install script should detect if it's executed as root and don't use sudo if true. Otherwise installation will fail with an error and I have to install sudo additionally in a docker container to install it as root.

Use case: Install tflint in a python container as part of [pre-commit](https://github.com/gruntwork-io/pre-commit) hooks to lint in CI.

## Version

```console
$ curl https://raw.githubusercontent.com/terraform-linters/tflint/master/install_linux.sh | bash
```
